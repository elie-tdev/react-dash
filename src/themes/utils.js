export const opacity = (hex, opacity) => {
  var result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex);
  if (!result) {
    throw new Error('Invalid hex', hex);
  }
  return `rgba(${parseInt(result[1], 16)}, ${parseInt(result[2], 16)}, ${parseInt(
    result[3],
    16,
  )}, ${opacity})`;
};

export const gradient = ({ type = 'linear', colors, background }) =>
  `${type}-gradient(${colors.join(', ')}), ${background};`;
