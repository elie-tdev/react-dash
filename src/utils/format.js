const formatter = new Intl.NumberFormat('en-US', {
  style: 'currency',
  currency: 'USD',
  minimumFractionDigits: 2,
});

const parseFloat = (str, val = 2) => {
  str = str && str.toString();
  str = str && str.indexOf('.') > -1 ? str.slice(0, str.indexOf('.') + val + 1) : str;
  return Number(str);
};

export const formatDollar = number => formatter.format(parseFloat(number, 2));
