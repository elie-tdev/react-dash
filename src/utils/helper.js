const maskAccountNumber = accNumber => {
  return accNumber && <>&bull;&bull;&bull;&bull; {accNumber.slice(accNumber.length - 4)}</>;
};

export default maskAccountNumber;
