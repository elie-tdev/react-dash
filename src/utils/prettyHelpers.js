export const prettyCurrency = (amount, currency = 'USD') => {
  if (amount == null) {
    return false;
  }
  if (amount < 0) {
    amount = 0;
  }
  return amount.toLocaleString('en', { style: 'currency', currency });
};

export const prettyDateTime = time => {
  if (time == null) {
    return false;
  }
  let d = new Date(time);
  return (
    d.toLocaleDateString('en', {
      month: '2-digit',
      day: '2-digit',
      year: '2-digit',
    }) +
    ' at ' +
    d.toLocaleTimeString('en', { hour: '2-digit', minute: '2-digit' }).replace(/^0(?:0:0?)?/, '')
  );
};

export const prettyDate = time => new Date(time).toLocaleDateString('en', {});

export const prettyTime = time =>
  new Date(time).toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' });

export const prettyHour = time =>
  new Date(time)
    .toLocaleTimeString('en', { hour: '2-digit', minute: '2-digit' })
    .replace(/^0(?:0:0?)?/, '');

export const prettyPhone = number =>
  number.substr(2).replace(/(\d{1,2})(\d{1})?(\d{1,3})?(\d{1,4})?/, function (_, p1, p2, p3, p4) {
    let output = '';
    if (p1) output = `${p1}`;
    if (p2) output += `${p2}-`;
    if (p3) output += `${p3}-`;
    if (p4) output += `${p4}`;
    return output;
  });

export const minimumDays = (date, numDays) => {
  const d = new Date(date);
  const totalNumDays = daysIntoYear(d) + numDays;
  return Boolean(daysIntoYear(d) >= totalNumDays || totalNumDays <= 365);
};

export const daysIntoYear = date => {
  return (
    (Date.UTC(date.getFullYear(), date.getMonth(), date.getDate()) -
      Date.UTC(date.getFullYear(), 0, 0)) /
    24 /
    60 /
    60 /
    1000
  );
};

export const cleanAmountRegex = value => {
  return value.replace(/[^0-9]/g, '');
};

export const cleanAmount = value => {
  const pos = value.indexOf('.');
  let result;
  if (pos !== -1) {
    const part1 = value.substr(0, pos);
    const part2 = value.substr(pos + 1);
    result = cleanAmountRegex(part1) + '.' + cleanAmountRegex(part2);
  } else {
    result = cleanAmountRegex(value);
  }
  return result;
};
