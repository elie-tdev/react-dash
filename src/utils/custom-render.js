import React from 'react';
import { I18nextProvider } from 'react-i18next';
import { render } from '@testing-library/react';
import { CssBaseline } from '@material-ui/core';
import { BrowserRouter as Router } from 'react-router-dom';
import { ThemeProvider } from '@material-ui/core/styles';
import { MockedProvider } from '@apollo/client/testing';

import { MenuContextProvider } from '@/contexts';
import i18n from '@/translations';
import { LightTheme } from '@/themes';

const wrapper = ({ mocks = [], menuContextProviderProps = {} }) => {
  const Wrapper = ({ children }) => {
    return (
      <Router>
        <MockedProvider mocks={mocks} addTypename={false}>
          <ThemeProvider theme={LightTheme}>
            <MenuContextProvider {...menuContextProviderProps}>
              <CssBaseline />
              <I18nextProvider i18n={i18n}>{children}</I18nextProvider>
            </MenuContextProvider>
          </ThemeProvider>
        </MockedProvider>
      </Router>
    );
  };

  return Wrapper;
};

const customRender = (ui, { mocks, menuContextProviderProps, ...restOptions } = {}) =>
  render(ui, { wrapper: wrapper({ mocks, menuContextProviderProps }), ...restOptions });

// re-export everything
export * from '@testing-library/react';

// override render method
export { customRender as render };
