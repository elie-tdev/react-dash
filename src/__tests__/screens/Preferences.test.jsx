import React from 'react';

import Preferences from '@/screens/Preferences';
import { render } from '@/utils/custom-render';

describe('Preferences Screen', () => {
  test('renders Preferences screen', () => {
    const { container } = render(<Preferences />);
    expect(container).toMatchSnapshot();
  });
});
