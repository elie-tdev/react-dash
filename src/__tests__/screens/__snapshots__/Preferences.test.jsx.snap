// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preferences Screen renders Preferences screen 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root-32"
  >
    <div
      class="MuiBox-root MuiBox-root-54 makeStyles-headerContainer-35 makeStyles-row-8"
      data-testid="Header"
    >
      <div
        class="MuiContainer-root makeStyles-container-33"
      >
        <div
          class="MuiBox-root MuiBox-root-55 makeStyles-row-8 makeStyles-headerContentBox-36"
        >
          <div
            class="MuiBox-root MuiBox-root-56 makeStyles-menuIconContainer-42"
            data-testid="MenuIconContainer"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-menuIcon-41"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
              />
            </svg>
          </div>
          <div
            class="MuiBox-root MuiBox-root-57 makeStyles-logoBox-50 makeStyles-logoBox-53"
          >
            <img
              class="makeStyles-logo-51"
              src="logo.png"
            />
          </div>
          <h2
            class="MuiTypography-root makeStyles-title-34 makeStyles-title-52 MuiTypography-h2"
          >
            Customer Dashboard
          </h2>
        </div>
        <div
          class="MuiBox-root MuiBox-root-64 makeStyles-column-10"
          data-testid="LoanSelectButton"
        />
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-65 makeStyles-headerSpacer-39"
    />
    <div
      class="MuiBox-root MuiBox-root-66 makeStyles-headerBottomWrapp-37 makeStyles-screenPaddingOn-21 makeStyles-row-8"
    >
      <div
        class="MuiContainer-root makeStyles-headerBottomContainer-38"
      >
        <div
          class="MuiBox-root MuiBox-root-67 makeStyles-row-8"
        >
          <div
            class="MuiBox-root MuiBox-root-77 makeStyles-row-8 makeStyles-desktopMenuItem-68 makeStyles-leftMenu-69"
            data-testid="NavigationMenuItem-desktop"
          >
            <div
              class="MuiBox-root MuiBox-root-78 makeStyles-iconContainer-72 makeStyles-column-10 makeStyles-icon-75"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root makeStyles-bottomMenuIcon-46"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 13h8V3H3v10zm0 8h8v-6H3v6zm10 0h8V11h-8v10zm0-18v6h8V3h-8z"
                />
              </svg>
            </div>
            <p
              class="MuiTypography-root makeStyles-itemName-73 MuiTypography-body1"
            >
              Loan Overview
            </p>
          </div>
          <div
            class="MuiBox-root MuiBox-root-79 makeStyles-row-8 makeStyles-desktopMenuItem-68 makeStyles-leftMenu-69"
            data-testid="NavigationMenuItem-desktop"
          >
            <div
              class="MuiBox-root MuiBox-root-80 makeStyles-iconContainer-72 makeStyles-column-10 makeStyles-icon-75"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root makeStyles-bottomMenuIcon-46"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M4 10h3v7H4zM10.5 10h3v7h-3zM2 19h20v3H2zM17 10h3v7h-3zM12 1L2 6v2h20V6z"
                />
              </svg>
            </div>
            <p
              class="MuiTypography-root makeStyles-itemName-73 MuiTypography-body1"
            >
              Balance Detail
            </p>
          </div>
          <div
            class="MuiBox-root MuiBox-root-81 makeStyles-row-8 makeStyles-desktopMenuItem-68 makeStyles-leftMenu-69"
            data-testid="NavigationMenuItem-desktop"
          >
            <div
              class="MuiBox-root MuiBox-root-82 makeStyles-iconContainer-72 makeStyles-column-10 makeStyles-icon-75"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root makeStyles-bottomMenuIcon-46"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M2.01 21L23 12 2.01 3 2 10l15 2-15 2z"
                />
              </svg>
            </div>
            <p
              class="MuiTypography-root makeStyles-itemName-73 MuiTypography-body1"
            >
              Payment History
            </p>
          </div>
          <div
            class="MuiBox-root MuiBox-root-83 makeStyles-row-8 makeStyles-desktopMenuItem-68 makeStyles-leftMenu-69"
            data-testid="NavigationMenuItem-desktop"
          >
            <div
              class="MuiBox-root MuiBox-root-84 makeStyles-iconContainer-72 makeStyles-column-10 makeStyles-icon-75"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root makeStyles-bottomMenuIcon-46"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 5v14H5V5h14m1.1-2H3.9c-.5 0-.9.4-.9.9v16.2c0 .4.4.9.9.9h16.2c.4 0 .9-.5.9-.9V3.9c0-.5-.5-.9-.9-.9zM11 7h6v2h-6V7zm0 4h6v2h-6v-2zm0 4h6v2h-6zM7 7h2v2H7zm0 4h2v2H7zm0 4h2v2H7z"
                />
              </svg>
            </div>
            <p
              class="MuiTypography-root makeStyles-itemName-73 MuiTypography-body1"
            >
              Statements
            </p>
          </div>
        </div>
        <div
          class="MuiBox-root MuiBox-root-85 makeStyles-row-8 makeStyles-menuItemsRightBox-44"
        >
          <div
            class="MuiBox-root MuiBox-root-86 makeStyles-row-8 makeStyles-desktopMenuItem-68 makeStyles-leftMenu-69"
            data-testid="NavigationMenuItem-desktop"
          >
            <div
              class="MuiBox-root MuiBox-root-87 makeStyles-iconContainer-72 makeStyles-column-10 makeStyles-icon-75"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root makeStyles-menuIcon-41"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19.14 12.94c.04-.3.06-.61.06-.94 0-.32-.02-.64-.07-.94l2.03-1.58c.18-.14.23-.41.12-.61l-1.92-3.32c-.12-.22-.37-.29-.59-.22l-2.39.96c-.5-.38-1.03-.7-1.62-.94l-.36-2.54c-.04-.24-.24-.41-.48-.41h-3.84c-.24 0-.43.17-.47.41l-.36 2.54c-.59.24-1.13.57-1.62.94l-2.39-.96c-.22-.08-.47 0-.59.22L2.74 8.87c-.12.21-.08.47.12.61l2.03 1.58c-.05.3-.09.63-.09.94s.02.64.07.94l-2.03 1.58c-.18.14-.23.41-.12.61l1.92 3.32c.12.22.37.29.59.22l2.39-.96c.5.38 1.03.7 1.62.94l.36 2.54c.05.24.24.41.48.41h3.84c.24 0 .44-.17.47-.41l.36-2.54c.59-.24 1.13-.56 1.62-.94l2.39.96c.22.08.47 0 .59-.22l1.92-3.32c.12-.22.07-.47-.12-.61l-2.01-1.58zM12 15.6c-1.98 0-3.6-1.62-3.6-3.6s1.62-3.6 3.6-3.6 3.6 1.62 3.6 3.6-1.62 3.6-3.6 3.6z"
                />
              </svg>
            </div>
            <p
              class="MuiTypography-root makeStyles-itemName-73 MuiTypography-body1"
            >
              Preferences
            </p>
          </div>
          <div
            class="MuiBox-root MuiBox-root-88 makeStyles-row-8 makeStyles-desktopMenuItem-68 makeStyles-leftMenu-69"
            data-testid="NavigationMenuItem-desktop"
          >
            <div
              class="MuiBox-root MuiBox-root-89 makeStyles-iconContainer-72 makeStyles-column-10 makeStyles-icon-75"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root makeStyles-menuIcon-41"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M10.09 15.59L11.5 17l5-5-5-5-1.41 1.41L12.67 11H3v2h9.67l-2.58 2.59zM19 3H5c-1.11 0-2 .9-2 2v4h2V5h14v14H5v-4H3v4c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                />
              </svg>
            </div>
            <p
              class="MuiTypography-root makeStyles-itemName-73 MuiTypography-body1"
            >
              Log Out
            </p>
          </div>
        </div>
        <div
          class="MuiBox-root MuiBox-root-90 makeStyles-menuIconRightBox-45"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root makeStyles-menuIcon-41"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
            />
          </svg>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-91 makeStyles-headerBottomSpacer-40"
    />
    <div
      class="MuiBox-root MuiBox-root-92 makeStyles-screenPaddingOff-20 makeStyles-elevatedBox-30 makeStyles-withMinHeight-31"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
      >
        <div
          class="MuiGrid-root makeStyles-gridContainer-93 makeStyles-gridContainer-94 MuiGrid-container MuiGrid-spacing-xs-3"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
          >
            <h4
              class="MuiTypography-root makeStyles-groupTitle-102 MuiTypography-h4"
            >
              Preferences
            </h4>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-lg-4"
          >
            <div
              class="MuiPaper-root MuiCard-root makeStyles-cardBorder-3 makeStyles-card-96 makeStyles-card-111 MuiPaper-elevation2 MuiPaper-rounded"
            >
              <div
                class="MuiBox-root MuiBox-root-115 makeStyles-preferenceContainer-97 makeStyles-preferenceContainer-112"
              >
                <div
                  class="MuiBox-root MuiBox-root-116 makeStyles-iconAndTitleContainer-99 makeStyles-iconAndTitleContainer-113"
                >
                  <div
                    class="MuiBox-root MuiBox-root-117 makeStyles-iconContainer-105 makeStyles-iconContainer-114"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M21 18v1c0 1.1-.9 2-2 2H5c-1.11 0-2-.9-2-2V5c0-1.1.89-2 2-2h14c1.1 0 2 .9 2 2v1h-9c-1.11 0-2 .9-2 2v8c0 1.1.89 2 2 2h9zm-9-2h10V8H12v8zm4-2.5c-.83 0-1.5-.67-1.5-1.5s.67-1.5 1.5-1.5 1.5.67 1.5 1.5-.67 1.5-1.5 1.5z"
                      />
                    </svg>
                  </div>
                  <div
                    class="MuiBox-root MuiBox-root-118 makeStyles-titleContainer-100"
                  >
                    <p
                      class="MuiTypography-root makeStyles-preferenceTitle-103 MuiTypography-body1"
                    >
                      Payment Accounts
                    </p>
                    <p
                      class="MuiTypography-root makeStyles-preferenceSubtitle-104 MuiTypography-body1"
                    >
                      Add and edit your payment accounts
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-lg-4"
          >
            <div
              class="MuiPaper-root MuiCard-root makeStyles-cardBorder-3 makeStyles-card-96 makeStyles-card-119 MuiPaper-elevation2 MuiPaper-rounded"
            >
              <div
                class="MuiBox-root MuiBox-root-123 makeStyles-preferenceContainer-97 makeStyles-preferenceContainer-120"
              >
                <div
                  class="MuiBox-root MuiBox-root-124 makeStyles-iconAndTitleContainer-99 makeStyles-iconAndTitleContainer-121"
                >
                  <div
                    class="MuiBox-root MuiBox-root-125 makeStyles-iconContainer-105 makeStyles-iconContainer-122"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M18 8h-1V6c0-2.76-2.24-5-5-5S7 3.24 7 6v2H6c-1.1 0-2 .9-2 2v10c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V10c0-1.1-.9-2-2-2zm-6 9c-1.1 0-2-.9-2-2s.9-2 2-2 2 .9 2 2-.9 2-2 2zm3.1-9H8.9V6c0-1.71 1.39-3.1 3.1-3.1 1.71 0 3.1 1.39 3.1 3.1v2z"
                      />
                    </svg>
                  </div>
                  <div
                    class="MuiBox-root MuiBox-root-126 makeStyles-titleContainer-100"
                  >
                    <p
                      class="MuiTypography-root makeStyles-preferenceTitle-103 MuiTypography-body1"
                    >
                      Change Password
                    </p>
                    <p
                      class="MuiTypography-root makeStyles-preferenceSubtitle-104 MuiTypography-body1"
                    >
                      Update or reset your password
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-lg-4"
          >
            <div
              class="MuiPaper-root MuiCard-root makeStyles-cardBorder-3 makeStyles-card-96 makeStyles-card-127 MuiPaper-elevation2 MuiPaper-rounded"
            >
              <div
                class="MuiBox-root MuiBox-root-131 makeStyles-preferenceContainer-97 makeStyles-preferenceContainer-128"
              >
                <div
                  class="MuiBox-root MuiBox-root-132 makeStyles-iconAndTitleContainer-99 makeStyles-iconAndTitleContainer-129"
                >
                  <div
                    class="MuiBox-root MuiBox-root-133 makeStyles-iconContainer-105 makeStyles-iconContainer-130"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
                      />
                    </svg>
                  </div>
                  <div
                    class="MuiBox-root MuiBox-root-134 makeStyles-titleContainer-100"
                  >
                    <p
                      class="MuiTypography-root makeStyles-preferenceTitle-103 MuiTypography-body1"
                    >
                      Change of Address
                    </p>
                    <p
                      class="MuiTypography-root makeStyles-preferenceSubtitle-104 MuiTypography-body1"
                    >
                      Submit a change of address request
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-154 makeStyles-root-135"
      data-testid="Footer"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
      >
        <div
          class="MuiGrid-root makeStyles-gridContainer-93 makeStyles-gridContainer-155 makeStyles-mainGridContainer-146 MuiGrid-container"
        >
          <div
            class="MuiGrid-root makeStyles-desktopLogoContainer-145 MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-3"
          >
            <img
              class="makeStyles-logo-136 makeStyles-logo-156"
              src="logo.png"
            />
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-md-4 MuiGrid-grid-lg-3"
          >
            <div
              class="MuiBox-root MuiBox-root-157 makeStyles-menuBox-149"
            >
              <h4
                class="MuiTypography-root makeStyles-menuHeadTitle-137 MuiTypography-h4"
              >
                FreedomPlus
              </h4>
              <div
                class="MuiBox-root MuiBox-root-158 makeStyles-menuList-151"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-link-143 MuiTypography-colorPrimary"
                  href="https://www.freedomplus.com/privacypolicy/"
                  rel="noopener"
                  target="_blank"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root makeStyles-root-159 makeStyles-root-160 MuiButton-contained makeStyles-menuItemButton-144 MuiButton-containedSizeLarge MuiButton-sizeLarge MuiButton-disableElevation MuiButton-fullWidth"
                    data-testid="SecondaryButton"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <h4
                        class="MuiTypography-root makeStyles-menuItem-138 MuiTypography-h4"
                      >
                        Privacy Notice
                      </h4>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </a>
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-link-143 MuiTypography-colorPrimary"
                  href="https://www.freedomplus.com/california-privacy-notice/"
                  rel="noopener"
                  target="_blank"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root makeStyles-root-159 makeStyles-root-161 MuiButton-contained makeStyles-menuItemButton-144 MuiButton-containedSizeLarge MuiButton-sizeLarge MuiButton-disableElevation MuiButton-fullWidth"
                    data-testid="SecondaryButton"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <h4
                        class="MuiTypography-root makeStyles-menuItem-138 MuiTypography-h4"
                      >
                        California Privacy Notice
                      </h4>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </a>
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-link-143 MuiTypography-colorPrimary"
                  href="https://www.freedomplus.com/termsofuse"
                  rel="noopener"
                  target="_blank"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root makeStyles-root-159 makeStyles-root-162 MuiButton-contained makeStyles-menuItemButton-144 MuiButton-containedSizeLarge MuiButton-sizeLarge MuiButton-disableElevation MuiButton-fullWidth"
                    data-testid="SecondaryButton"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <h4
                        class="MuiTypography-root makeStyles-menuItem-138 MuiTypography-h4"
                      >
                        Terms of Use
                      </h4>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </a>
              </div>
            </div>
          </div>
          <hr
            class="MuiDivider-root makeStyles-divider-150"
          />
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-md-4 MuiGrid-grid-lg-3"
          >
            <div
              class="MuiBox-root MuiBox-root-163 makeStyles-menuBox-149"
            >
              <h4
                class="MuiTypography-root makeStyles-menuHeadTitle-137 MuiTypography-h4"
              >
                Consolidation Plus
              </h4>
              <div
                class="MuiBox-root MuiBox-root-164 makeStyles-menuList-151"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-link-143 MuiTypography-colorPrimary"
                  href="https://consolplus.com/privacy-notice/"
                  rel="noopener"
                  target="_blank"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root makeStyles-root-159 makeStyles-root-165 MuiButton-contained makeStyles-menuItemButton-144 MuiButton-containedSizeLarge MuiButton-sizeLarge MuiButton-disableElevation MuiButton-fullWidth"
                    data-testid="SecondaryButton"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <h4
                        class="MuiTypography-root makeStyles-menuItem-138 MuiTypography-h4"
                      >
                        Privacy Notice
                      </h4>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </a>
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-link-143 MuiTypography-colorPrimary"
                  href="https://consolplus.com/california-privacy-notice/"
                  rel="noopener"
                  target="_blank"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root makeStyles-root-159 makeStyles-root-166 MuiButton-contained makeStyles-menuItemButton-144 MuiButton-containedSizeLarge MuiButton-sizeLarge MuiButton-disableElevation MuiButton-fullWidth"
                    data-testid="SecondaryButton"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      <h4
                        class="MuiTypography-root makeStyles-menuItem-138 MuiTypography-h4"
                      >
                        California Privacy Notice
                      </h4>
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </a>
              </div>
            </div>
          </div>
          <hr
            class="MuiDivider-root makeStyles-divider-150"
          />
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-md-4 MuiGrid-grid-lg-3"
          >
            <div
              class="MuiBox-root MuiBox-root-167 makeStyles-menuBox-149"
            >
              <h4
                class="MuiTypography-root makeStyles-menuHeadTitle-137 MuiTypography-h4"
              >
                Mailing Address
              </h4>
              <div
                class="MuiBox-root MuiBox-root-168 makeStyles-menuList-151"
              >
                <h4
                  class="MuiTypography-root makeStyles-menuDescription-139 MuiTypography-h4"
                >
                  Main Office
                </h4>
                <h4
                  class="MuiTypography-root makeStyles-menuDescription-139 MuiTypography-h4"
                >
                  2114 E Rio Salado Blvd
                </h4>
                <h4
                  class="MuiTypography-root makeStyles-menuDescription-139 MuiTypography-h4"
                >
                  Tempe, AZ 85281
                </h4>
                <h4
                  class="MuiTypography-root makeStyles-menuDescription-139 MuiTypography-h4"
                >
                   PO Box 2340
                </h4>
                <h4
                  class="MuiTypography-root makeStyles-menuDescription-139 MuiTypography-h4"
                >
                  Phoenix, AZ 85002-2340
                </h4>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
      >
        <div
          class="MuiGrid-root makeStyles-gridContainer-93 makeStyles-gridContainer-169 MuiGrid-container MuiGrid-spacing-xs-3"
        >
          <div
            class="MuiGrid-root MuiGrid-item makeStyles-menuBottomWrap-140 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-12"
          >
            <div
              class="MuiBox-root MuiBox-root-170 makeStyles-bottomLogoBox-147"
            >
              <img
                class="makeStyles-logo-136 makeStyles-logo-171"
                src="logo.png"
              />
            </div>
            <div
              class="MuiBox-root MuiBox-root-172 makeStyles-bottomCopyrightBox-148"
            >
              <h4
                class="MuiTypography-root makeStyles-menuBottomDescription-141 MuiTypography-h4"
              >
                © 2021 Freedom Financial Asset Management, LLC. All Rights Reserved.
              </h4>
            </div>
            <button
              class="MuiButtonBase-root MuiButton-root makeStyles-root-159 makeStyles-root-173 MuiButton-contained makeStyles-button-142 MuiButton-containedSizeLarge MuiButton-sizeLarge MuiButton-disableElevation MuiButton-fullWidth"
              data-testid="SecondaryButton"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                Contact Us
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
